blueprints/app/files/tests/helpers/setup-application-test.js

```diff
@@ -0,0 +1,5 @@
+import { setupApplicationTest } from 'ember-qunit';
+
+export default function(hooks) {
+  setupApplicationTest(hooks);
```

Comment:
This seems like a good place to put a comment telling users why this file exists and what to do here, similar to [this](https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/ember-cli-build.js#L10). Good opportunity to educate, and also avoid confusion over this apparently extraneous file.